packages/common/src/hooks/purchaseContent/utils.ts

```diff
@@ -34,12 +36,15 @@ export const getExtraAmount = ({
   return extraAmount
 }
 
-export const isTrackStreamPurchaseable = (
-  track: UserTrackMetadata
-): track is PurchaseableTrackStreamMetadata =>
-  isContentUSDCPurchaseGated(track.stream_conditions)
+export const isStreamPurchaseable = (
+  metadata: PurchaseableContentMetadata
+): metadata is
+  | PurchaseableTrackStreamMetadata
+  | PurchaseableAblumStreamMetadata =>
+  isContentUSDCPurchaseGated(metadata.stream_conditions)
 
-export const isTrackDownloadPurchaseable = (
-  track: UserTrackMetadata
-): track is PurchaseableTrackDownloadMetadata =>
-  isContentUSDCPurchaseGated(track.download_conditions)
+export const isDownloadPurchaseable = (
```

Comment:
should we rename this back to `isTrackDownloadPurchaseable` since this is only for tracks right now?